aospy/test/test_utils_times.py

```diff
@@ -530,5 +534,29 @@ def test_yearly_average_masked_data():
     xr.testing.assert_allclose(actual, desired)
 
 
-if __name__ == '__main__':
-    sys.exit(unittest.main())
+def test_average_time_bounds():
```

Comment:
Regardless of whether or not you switch to a float-only method, could you add a test for the use case where floats are used rather than dates, since that is the use case in `data_loader.py`?